roles/config-quay-enterprise/tasks/main.yml

```diff
@@ -128,6 +128,13 @@
 - name: Include firewall tasks
   include_tasks: firewall.yml
 
+- name: Stop quay to complete setup
+  systemd:
+    name: "{{ quay_name }}"
+    state: stopped
+
 - name: Setup Initial User and Configuration
   include_tasks: complete_setup.yml
```

Comment:
This playbook cannot be run on a single machine as near the conclusion, it applys the final configuration file which is needed to tell quay the setup process is complete